File: other_lang/Java/table.java
import java.util.*;
import java.util.Scanner;




public class table
{

	public static void main(String[] args)
	{
		Scanner scn = new Scanner(System.in);
		System.out.println("Enter the number : ");
		int n = scn.nextInt();
		int m;

		//21BCS8908__Avinash__Kumar
		for(int i=1;i<=10;i++)
		{
			m=n*i;
			System.out.println(n+" x " +i+ " = " +m );
		}
}
}

File: other_lang/Python/table.py
## PARAMETERIZED FUNCTION


def print_table_of(num):
    for i in range(1,11):
        print(num,' x ', i, ' = ',num*i)


def main():
    while True:
        input_num = int(input("Please Enter A Number To Print Its Multiplication Table: "))
        if input_num in range(2,21):
            for table_num in range(2,input_num + 1):
                print(f"\nTable Of {table_num}")
                print_table_of(table_num)

        else:
            print("Please Enter Table In range Of 2 to 20 Only.")

        print("")



main()
